app/assets/stylesheets/application.scss

```diff
@@ -24,8 +24,17 @@ main {
   @include inner-block;
 }
 
+.print-page {
+  display: none;
+  margin-top:$gutter;
+
+  .js-enabled & {
+    display: block;
+  }
+}
```

Comment:
Is there any reason this isn't nested within the `#manuals-frontend` selector? It would fit  better with the approach the rest of the Sass has taken in this file if it was nested.